### ISSUE TITLE
Custom end date before start date still returns a result!

### ISSUE THREAD

**Issue by sabahfromlondon** [open-contracting/data-registry#77]
If I select the start date: 22 Jun 2020
And end date 21 Jun 2019

The registry returned results, where it should have thrown an error. See screenshot.

The fields should really be named "Start date" and "End date" as well.

<img width="818" alt="Screenshot 2021-06-17 164547" src="https://user-images.githubusercontent.com/11212467/122430429-81cc8480-cf8b-11eb-90d9-1f1f9ac22ff4.png">

**Comment by hrubyjan** [comment]
Disable selection of nonsensical values and rename fields

**Comment by t-mrazek** [comment]
Fixed, the date fields are restricted by the selection of a date in the opposite field. The reset button added for the possibility of a value removing.

**Comment by sabahfromlondon** [comment]
Yes look good to me now. I think it would also be helpful if the user is not allowed to select a date in the future. Is this possible?
